gammapy/data/pointing.py

```diff
@@ -151,6 +206,87 @@ def altaz(self):
 
         return self.radec.transform_to(frame)
 
+    @lazyproperty
+    def fixed_altaz(self):
+        """The fixed coordinates in AltAz of the observation.
+
+        None if not a DRIFT observation
+        """
+        if self.mode != PointingMode.DRIFT:
+            return None
+
+        alt = u.Quantity(self.meta["ALT_PNT"], u.deg)
+        az = u.Quantity(self.meta["AZ_PNT"], u.deg)
+        return SkyCoord(alt=alt, az=az, frame=self.altaz_frame)
+
+    @lazyproperty
+    def fixed_icrs(self):
+        """
+        The fixed coordinates in ICRS of the observation.
+
+        None if not a POINTING observation
+        """
+        if self.mode != PointingMode.POINTING:
+            return None
+
+        return self.radec
+
+    def get_icrs(self, obstime):
```

Comment:
Could make the name here a bit more explicit, like `get_icrc_at_time()` or similar...

Reply:
Since the method has a single positional argument called obstime, I don't really see what:
```
pointing.get_icrs_at_time(obstime)
```
offers over:
```
pointing.get_icrs(obstime)
```